Purchases/Caching/RCDeviceCache.m

```diff
@@ -54,40 +56,35 @@ - (instancetype)initWith:(NSUserDefaults *)userDefaults
 
 #pragma mark - appUserID
 
-- (nullable NSString *)cachedLegacyAppUserID
-{
```

Comment:
format for whether the `{` should be in the method definition line or the next one is sort of mixed. We seem to have more of `{` on the same line, so I'm going with standardizing to that.